src/get-started/learn-more.md

```diff
@@ -1,54 +1,163 @@
-* [Building layouts][] tutorial
-* [Add interactivity][] tutorial
-
-## Apply your existing knowledge
-
-* [Flutter for Android developers][]
-* [Flutter for SwiftUI developers][]
-* [Flutter for UIKit developers][]
-* [Flutter for React Native developers][]
-* [Flutter for web developers][]
-* [Flutter for Xamarin.Forms developers][]
-
-## Other resources
-
-* [Flutter samples][]
-* [Flutter cookbook][]
-* [Bootstrap into Dart: learn more about the language][]
-* [Flutter API Docs][]
-* [The Complete Flutter Developer Bootcamp Using Dart][]
-* [Udacity online Flutter training][]
-
-Reach out to us at our [mailing list][]. We'd love to hear from you!
-
-Happy Fluttering!
-
-[Add interactivity]: {{site.url}}/ui/interactivity
-[Bootstrap into Dart: learn more about the language]: {{site.url}}/resources/bootstrap-into-dart
-[Building layouts]: {{site.url}}/ui/layout/tutorial
-[The Complete Flutter Developer Bootcamp Using Dart]: https://www.appbrewery.co/p/flutter-development-bootcamp-with-dart
-[Flutter API Docs]: {{site.api}}
-[Flutter cookbook]: {{site.url}}/cookbook
-[Flutter for Android developers]: {{site.url}}/get-started/flutter-for/android-devs
-[Flutter for SwiftUI developers]: {{site.url}}/get-started/flutter-for/swiftui-devs
-[Flutter for UIKit developers]: {{site.url}}/get-started/flutter-for/uikit-devs
-[Flutter for React Native developers]: {{site.url}}/get-started/flutter-for/react-native-devs
-[Flutter samples]: https://flutter.github.io/samples
-[Flutter for web developers]: {{site.url}}/get-started/flutter-for/web-devs
-[Flutter for Xamarin.Forms developers]: {{site.url}}/get-started/flutter-for/xamarin-forms-devs
-[Introduction to Flutter's UI]: {{site.url}}/ui
-[mailing list]: mailto:{{site.email}}
-[Udacity online Flutter training]: https://www.udacity.com/course/build-native-mobile-apps-with-flutter--ud905
+## Layout
+
+Given that Flutter is a UI toolkit, you’ll spend a lot of time creating layouts
+with Flutter widgets. In this section, you’ll learn how to build simple and
+complex layouts in Flutter with some of the most common layout widgets in
+Flutter. You’ll also learn about how to use DevTools to understand how your apps
+are being laid out, and you’ll learn how to debug one of the most common Flutter
+errors, which is related to the most common Flutter layout widgets.
+
+### Introduction to layouts
+
+The following tutorial will introduce you to layouts in Flutter, and give you an
+opportunity to work with the most common layout widgets.
+
+1. <i class="material-icons" aria-hidden="true">bolt</i> Take action! - Tutorial: [Layouts in Flutter][]
+
+### Constraints
+
+Understanding “constraints” in Flutter are an important part of understanding
+how Flutter’s layout algorithm works.
+
+The following resources will teach you what constraints are, some widgets that
+allow you to work with constraints, and how to deal with “unbounded constraints”
+type errors.
+
+1. Article: [Understanding Constraints][]
```

Comment:
Style question: should these be ordered lists, or would bullets work just as well (e.g. "all things being equal we recommend going top-down on these four things, but it's not required")?